Makefile

```diff
@@ -70,7 +70,9 @@ build-all:
 	go build cmd/receptor.go
 
 test: receptor
-	@go test ./... -p 1 -parallel=16 -count=1
+	@echo "Running receptor tests" && \
+	go test -v ./... -p 1 -parallel=16 -count=1 | tee test-output.log | grep FAIL ; \
+	echo "See test-output.log or test-junit.xml for a more comprehensive report"
```

Comment:
If I understand correctly, test-junit.xml isn't being created by anything in the Makefile - it's only a thing that happens in the GitHub Actions container.  So it probably shouldn't be mentioned here.